core/src/main/scala/cats/Reducible.scala

```diff
@@ -112,6 +112,12 @@ import simulacrum.typeclass
       val F = self
       val G = Reducible[G]
     }
+
+  def minimum[A](fa: F[A])(implicit A: Order[A]): A =
```

Comment:
@johnynek do you have any further thoughts on this after my note about the syntax enrichment that currently exists?

If there's a strong preference for `min` then I can change it, but I do have a hesitation about potential refactoring bugs that could come out of it.